content/venue/accommodation.md

```diff
@@ -11,4 +11,50 @@ contacting the providers directly.
 
 Some options for accommodation in Mostar are as follows
 
-**TBD**
+## 1. Student Center Mostar
+   
+- Prices range from €12.78 without breakfast to €14.82 with breakfast
+- General Contact: **[**info@scm.ba**](mailto:info@scm.ba)**
+- Website: **[**www.scm.ba/hostel**](https://www.scm.ba/hostel/)**
```

Comment:
please use hugo extlink shortcode for all external links (forces target=blank), i.e.:

```
**{{< extlink href="https://www.scm.ba/hostel/" title="www.scm.ba/hostel" >}}**
```